.github/workflows/flowzone.yml

```diff
@@ -1524,4 +1524,7 @@ jobs:
             exit 1
           fi
 
-          echo ::set-output name=result::"${result}"
+          DELIMITER=$(echo $RANDOM | md5sum | head -c 32)
```

Comment:
The random delimiter is to avoid issues if the delimiter happens to be contained in the output